pkg/autoscaler/config/config.go

```diff
@@ -150,6 +154,7 @@ func NewConfigFromMap(data map[string]string) (*Config, error) {
 
 		cm.AsInt32("initial-scale", &lc.InitialScale),
 		cm.AsInt32("max-scale", &lc.MaxScale),
+		cm.AsInt32("max-max-scale", &lc.MaxMaxScale),
```

Comment:
I'm not sure whether we can have a better name :)

Reply:
Yes 😂

Reply:
MaxestScale 👅